src/components/SidePanels/DetailsPanel/RegionEditor.tsx

```diff
@@ -157,17 +160,17 @@ const RegionProperty: FC<RegionPropertyProps> = ({
 
 interface RegionInputProps extends InputHTMLAttributes<HTMLInputElement>  {
   type: HTMLInputTypeAttribute;
-  onChange?: (newValue: any) => void;
+  onChangeValue?: (newValue: any, originalValue: any) => void;
 }
 
 const RegionInput: FC<RegionInputProps> = ({
-  onChange,
+  onChangeValue,
   type,
   value,
   ...props
```

Comment:
this is sort of nitpicky but lets add step here, so we can just refrence it later instead of refrencing it as `props.step`